tests/D2L.CodeStyle.Analyzers.Test/Specs/RequireNamedArgumentsAnalyzer.cs

```diff
@@ -14,23 +14,23 @@ private class Thing {
 		public static void _arg2( int a1, int a2 ) { }
 		public static void _arg3( int a1, int a2, int a3 ) { }
 		public static void _arg4( int a1, int a2, int a3, int a4 ) { }
+		public static void _arg6( int a1, int a2, int a3, int a4, int a5, int a6 ) { }
 
 		// These will shrink as we shrink the max *blush*
+		public static void _arg7( int a1, int a2, int a3, int a4, int a5, int a6, int a7 ) { }
 		public static void _arg8( int a1, int a2, int a3, int a4, int a5, int a6, int a7, int a8 ) { }
-		public static void _arg9( int a1, int a2, int a3, int a4, int a5, int a6, int a7, int a8, int a9 ) { }
-		public static void _arg10( int a1, int a2, int a3, int a4, int a5, int a6, int a7, int a8, int a9, int a10 ) { }
 
 		public static void funcWithParams( int a, int b, int c, params int[] ps ) { }
 
 		public delegate void delegate0Args();
 		public delegate void delegate1Args( int a1 );
-		public delegate void delegate9Args( int a1, int a2, int a3, int a4, int a5, int a6, int a7, int a8, int a9 );
+		public delegate void delegate7Args( int a1, int a2, int a3, int a4, int a5, int a6, int a7);

```

Comment:
I'll fix this horrible white space mistake when I reduce it to 5.

Reply:
Haha, no big deal. Maybe it's testing that our analyzer works with horrible whitespace! ;)